lib/python/rosie/svn_pre_commit.py

```diff
@@ -65,6 +65,22 @@ def _get_access_info(self, info_node):
         access_list.sort()
         return owner, access_list
 
+    def _get_author_aliases(self, repos, txn=None):
+        """Return the current author alternative name map."""
+        txn_args = ()
+        if txn is not None:
+            txn_args = ("-t", txn)
+        try:
+            text = self._svnlook(
+                "cat", repos, "/R/O/S/I/E/trunk/author_aliases", *txn_args)
+        except Exception:
+            return {}
```

Comment:
FYI there is no pre-commit check for the validity of this file/property any more - aliases would just not work.